services/explorer/src/components/BlockTable/BlockTable.tsx

```diff
@@ -0,0 +1,242 @@
+import React from 'react';
+import {
+  Table,
+  TableHead,
+  TableRow,
+  TableCell,
+  TableBody,
+  Box,
+  TablePagination,
+  TableContainer,
+  ThemeProvider,
+  CssBaseline,
+} from '@mui/material';
+import { Label, Button, Icons } from '@taraxa_project/taraxa-ui';
+import type { TransactionData } from '../../models/TransactionData';
+import { TransactionStatus } from '../../models/TableData';
+import { theme } from '../../theme-provider';
+import { AddressLink } from '../Links';
+
+export interface TransactionDataItem extends TransactionData {
+  txHash: string;
+  dagLevel: string;
+  dagHash: string;
+}
+
+export const statusToLabel = (state: TransactionStatus) => {
+  if (state === TransactionStatus.SUCCESS) {
+    return (
+      <Label
+        variant='success'
+        label='Success'
+        gap
+        icon={<Icons.GreenCircledCheck />}
+      />
+    );
+  }
+  if (state === TransactionStatus.FAILURE) {
+    return (
+      <Label
+        variant='error'
+        label='Failure'
+        gap
+        icon={<Icons.RedCircledCancel />}
+      />
+    );
+  }
+  return (
+    <Label
+      variant='error'
+      label='Failure'
+      gap
+      icon={<Icons.RedCircledCancel />}
+    />
+  );
+};
+
+export interface BlockTableProps {
+  blockData: TransactionDataItem[];
+  onFilter?: () => void;
+  onDAGFilter?: () => void;
+  onPBFTFilter?: () => void;
+}
+
+export const BlockTable: React.FC<BlockTableProps> = (props) => {
```

Comment:
Would have been prettier to extract the props directly here, and without giving them a default value.
`export const BlockTable: React.FC<BlockTableProps> = ({blockData , onFilter, onDAGFilter , onPBFTFilter }: BlockTableProps =>`